config/mcu_definitions/main/BTT_SKR_Pico_v1.0.cfg

```diff
@@ -0,0 +1,25 @@
+[board_pins mcu_manufacturer]
+aliases:
+    MCU_X_STEP=gpio11 , MCU_X_DIR=gpio10 , MCU_X_EN=gpio12 ,
+    MCU_Y_STEP=gpio6 , MCU_Y_DIR=gpio5  , MCU_Y_EN=gpio7 ,
+    MCU_Z_STEP=gpio19  , MCU_Z_DIR=gpio28  , MCU_Z_EN=gpio2 ,
+    MCU_E0_STEP=gpio14  , MCU_E0_DIR=gpio13  , MCU_E0_EN=gpio15 ,
+    MCU_TMCUART=gpio9 , MCU_TMCTX=gpio8  ,
+
+    MCU_X-STOP=gpio4   , MCU_Y-STOP=gpio3  , MCU_Z-STOP=gpio25   ,
+    MCU_E0-STOP=gpio16 , 
+
+    MCU_HE=gpio23 ,
+    MCU_HB=gpio21 ,
+    MCU_LASER=gpio20 ,
```

Comment:
Be careful the gpio20 is a shared pin between LASER and FAN3.
As Klippain is made for 3D printers mainly, I would suggest keeping FAN3 and commenting this line (to know that it's here but not used)

Reply:
why bother for laser then?

Reply:
Yeah, you can remove it